frontend/src/metabase/query_builder/components/notebook/steps/SummarizeStep.tsx

```diff
@@ -13,14 +13,8 @@ function SummarizeStep({
   step,
   ...props
 }: NotebookStepUiComponentProps) {
-  const isMetric = step.question.type() === "metric";
```

Comment:
Some leftover from the integration branch. This change never worked and needs to be implemented differently.